src/commands/info/releasenotes/display.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// Needed this to ensure the "helpers" were decalred before read in examples
+/* eslint-disable @typescript-eslint/member-ordering */
+
+import { Env } from '@salesforce/kit';
+import { flags, SfdxCommand } from '@salesforce/command';
+import { getString } from '@salesforce/ts-types';
+import { Messages } from '@salesforce/core';
+
+import { getInfoConfig, InfoConfig } from '../../../shared/get-info-config';
+import { getReleaseNotes } from '../../../shared/get-release-notes';
+import { getDistTagVersion } from '../../../shared/get-dist-tag-version';
+
+// Initialize Messages with the current plugin directory
+Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is sfdxusing the messages framework can also be loaded this way.
+const messages = Messages.loadMessages('@salesforce/plugin-info', 'display');
+
+export default class Display extends SfdxCommand {
+  private static helpers = ['stable', 'stable-rc', 'latest', 'latest-rc', 'rc'];
+
+  public static description = messages.getMessage('commandDescription');
+
+  public static aliases = ['whatsnew'];
+
+  public static examples = [
```

Comment:
we normally put the examples in the messages file
ex: https://github.com/salesforcecli/plugin-user/blob/fd61fb24d9022bf79fcc5e63d558c84d79c18142/messages/create.json#L3

Reply:
note that they don't have the `$` in those